examples/morpheusvm/registry/registry.go

```diff
@@ -25,12 +25,12 @@ func init() {
 	errs := &wrappers.Errs{}
 	errs.Add(
 		// When registering new actions, ALWAYS make sure to append at the end.
-		Action.Register((&actions.Transfer{}).GetTypeID(), actions.UnmarshalTransfer),
+		Action.Register(&actions.Transfer{}, nil),
```

Comment:
as per other comment, let's keep `actions.UnmarshalFrom` so we don't have commented out code